### PR TITLE
Expose check-url configuration and disable client-side check if not enabled

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR disable client-side checks if server-side is not enabled (and so does not produce 500 errors)

### CODE REVIEW

File: js/config.js [hunk 1]
```diff
@@ -63,6 +63,11 @@ export const title = _meta('site-title');
 export const csrf_token = _meta('csrf-token');
 
 /**
+ * Is the check url feature enabled ?
+ */
+export const check_urls = JSON.parse(_meta('check-urls'));
```
Comment (davidbgk) [inline js/config.js]: Oh nice, I didn't think about using `JSON.parse` for that in territories, I'll update :)

File: js/dataset/display.js [hunk 2]
```diff
@@ -48,36 +48,38 @@ function prepare_resources() {
         });
 
         // Check asynchronuously the status of displayed resources
-        $this.find('.format-label').each(function() {
```
Comment (davidbgk) [inline js/dataset/display.js]: Did you change anything except the indent? Not sure if that's github messing up with the diff or if there are actual changes.

File: udata/templates/metadata.html [hunk 3]
```diff
@@ -41,4 +41,5 @@
 />
 {% endif %}
 
-<meta name="notify-in" content="{{ meta.notify_in or '#notifications-container' }}">
+<meta name="notify-in" content="{{ meta.notify_in or '#notifications-container' }}" />
+<meta name="check-urls" content="{{ 'true' if config.CROQUEMORT else 'false' }}" />
```
Comment (davidbgk) [inline udata/templates/metadata.html]: Maybe `str(bool(config.CROQUEMORT)).lower()` mmh typing it I'm not sure anymore it's more explicit…

**Comment by davidbgk** [comment]
r+

File: js/config.js (hunk 1 above)
Comment (noirbizarre) [inline js/config.js]: JSON.parse for primitive types works well ;)

File: js/dataset/display.js (hunk 2 above)
Comment (noirbizarre) [inline js/dataset/display.js]: Only the indent, not anything else.

File: udata/templates/metadata.html (hunk 3 above)
Comment (noirbizarre) [inline udata/templates/metadata.html]: :) I tried different syntax. Maybe I should create a filter `boolify` ?

Comment (davidbgk) [inline udata/templates/metadata.html]: Nevermind, the current version is fine.

**Pull request merged**